.env.sample

```diff
@@ -9,4 +9,10 @@ DB_USER_TEST = userName
 DB_PASSWORD_TEST = password
 DB_HOST_TEST = host
 DB_PORT_TEST = port
-DB_NAME_TEST = testDbName
\ No newline at end of file
+DB_NAME_TEST = testDbName
+
+CLIENT_ID = clintID
+CLIENT_SECRET = clientSecret
+CALLBACK_URL = http://localhost:3000/api/v1/users/auth/google/callback
```

Comment:
@Habinezajanvier can you remove the value from `.env.sample`, here environment variables are being exposed.